docker-compose/scripts/run.sh

```diff
@@ -2,26 +2,29 @@
 
 set -e
 
+log () {
+  echo "[$(date --rfc-3339 seconds)] - $1"
+}
+
 elastic_host=${kuzzle_services__db__client__host:-http://elasticsearch:9200}
 
-echo "[$(date --rfc-3339 seconds)] - Waiting for elasticsearch to be available"
+log "Waiting for elasticsearch"
 while ! curl -f -s -o /dev/null "$elastic_host"
 do
-    echo "[$(date --rfc-3339 seconds)] - Still trying to connect to $elastic_host"
+    log "Still trying to connect to $elastic_host"
     sleep 1
 done
 # create a tmp index just to force the shards to init
 curl -XPUT -s -o /dev/null "$elastic_host/%25___tmp"
-echo "[$(date --rfc-3339 seconds)] - Elasticsearch is up. Waiting for shards to be active (can take a while)"
+log "Elasticsearch is up. Waiting for shards..."
 E=$(curl -s "$elastic_host/_cluster/health?wait_for_status=yellow&wait_for_active_shards=1&timeout=60s")
 curl -XDELETE -s -o /dev/null "$elastic_host/%25___tmp"
 
 if ! (echo ${E} | grep -E '"status":"(yellow|green)"' > /dev/null); then
-    echo "[$(date --rfc-3339 seconds)] - Could not connect to elasticsearch in time. Aborting..."
+    log "Could not connect to elasticsearch in time. Aborting..."
     exit 1
 fi
 
-echo "[$(date --rfc-3339 seconds)] - Starting Kuzzle..."
+log "Starting Kuzzle..."
 
-pm2 start --silent /config/pm2.json
-pm2 logs --lines 0 --raw
+exec ./bin/kuzzle "$@"
```

Comment:
As current script is now copied into /usr/local/bin/kuzzle (according to your Dockerfile's changes), and outside application directory, shouldn't we call `/var/app/bin/kuzzle` (with absolute path) instead ?

Reply:
The path the script is located does not matter, only the one where it is run, being the container WORKDIR (/var/app), so everything should be OK (and is, tested locally)